.circleci/deployment/jobs.yml

```diff
@@ -133,6 +142,10 @@
     executor: docker-executor
     working_directory: ~/tdp-deploy
     steps:
+      - init-deploy
+      - build-and-tag-images:
+          backend-appname: tdp-backend-prod
+          frontend-appname: tdp-frontend-prod
```

Comment:
As this is under the `deploy-production` job, will we run into issues of this running from the HHS CircleCI? Will the `CIRCI_DOCKER_LOGIN` variable would need to be recreated there?